src/workers/layouts/horizontal.ts

```diff
@@ -54,3 +58,18 @@ function getHorizontalTimeLayout({ data, horizontalSettings }: ClientLayoutMessa
   return layout
 }
 
+function getHorizontalLeftAlignedLayout({ data, horizontalSettings }: ClientLayoutMessage): HorizontalLayout {
+  const scale = horizontalScaleFactory(horizontalSettings)
+  const layout: HorizontalLayout = new Map()
+
+  const firstEntryStartTime = data.nodes.entries().next().value[1].start_time
```

Comment:
Could this just use the start time of the run itself? `data.start_time`

Reply:
ugg... totally. haha updating